src/routes/public/images-router.js

```diff
@@ -3,7 +3,7 @@ var express = require('express');
 var router = express.Router();
 const imagesController = require('../../controllers/index').imagesController;
 const routes = require('../../utils/routes-values').IMAGES_ROUTS;
-const asyncMiddleware = require('../../utils/error-catcher').use;
+const asyncMiddleware = require('../../utils/error-catcher').asyncMiddleware;
```

Comment:
I think you can write such imports like 
const { asyncMiddleware } = require(...)
It's extensible too, in cases if you'd like to import multiple functions.